src/main/java/com/dataloom/datastore/pods/AuthorizationsMvcPod.java

```diff
@@ -0,0 +1,71 @@
+package com.dataloom.datastore.pods;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.web.servlet.config.annotation.ContentNegotiationConfigurer;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurationSupport;
+
+import com.dataloom.data.DataApi;
+import com.dataloom.datastore.authorization.controllers.AuthorizationsController;
+import com.dataloom.datastore.constants.CustomMediaType;
+import com.dataloom.datastore.converters.CsvHttpMessageConverter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ryantenney.metrics.spring.config.annotation.EnableMetrics;
+
+@Configuration
+@ComponentScan(
+    basePackageClasses = { AuthorizationsController.class },
+    includeFilters = @ComponentScan.Filter(
+        value = { org.springframework.stereotype.Controller.class,
+                org.springframework.web.bind.annotation.ControllerAdvice.class },
+        type = FilterType.ANNOTATION ) )
+@EnableAsync
+@EnableMetrics(
+    proxyTargetClass = true )
+public class AuthorizationsMvcPod extends WebMvcConfigurationSupport {
```

Comment:
Is this not considered a separate service? I'm seeing that different services in datastore are getting their own mvc pod right now